Makefile

```diff
@@ -16,6 +16,22 @@ test:
 	$(MAKE) -C pkg/policy-compiler test
 	$(MAKE) -C manager test
 
+.PHONY: run-integration-tests
```

Comment:
I'm not sure if a bash script in `hack` would be better to run the test instead of putting more code into the make file.

Reply:
this is an issue of taste, my personal take is that having a script from hack calling back into the make system creates a complex look (notice that all the other scripts are self-contained in that respect). so the natural place for such a sequence is in the makefile itself, as to its' length, this will reduce as we improve and current acceptable complexity of setting up m4d (I am working on that separately). for example: the `secret-provider` steps need to be merged into the m4d core deployment (and thus will go), I have a big problem with the architecture of test/services and think this code should be integrated as a mock code within the connectors (and thus will go), etc.

Reply:
I agree with @froesef that a script would be cleaner here, even if it's just a matter of taste. I also find the exports confusing, at least I wasn't aware of this kind of syntax.